README.md

```diff
@@ -14,7 +14,7 @@ Technically it consists of a single binary `lxqt-runner`. The binary is run in a
 
 ### Compiling source code
 
-Runtime dependencies are muparser, KWindowSystem, menu-cache and [lxqt-globalkeys](https://github.com/lxqt/lxqt-globalkeys).  
+Runtime dependencies are [lxqt-globalkeys](https://github.com/lxqt/lxqt-globalkeys), muparser, KWindowSystem, and optionally menu-cache.  
```

Comment:
I'd add something like:

**Note**: As `lxqt-globalkeysd` is not supported yet on wayland a shortcut for `lxqt-runner` has to be added in the compositor settings.